changelog/README.md

```diff
@@ -0,0 +1,15 @@
+The `changelog` directory contains the description of the changes introduced into the repository.
+The changes are essentially divided into 4 categories
+- changes: List of PRs brings Changes and/or Enhancements
+- bugfixes: List of PRs fixing existing issues
+- security: List of PRs fixing security issues
+- updates: Package Updates
+
+## How to add the file
+
+Based on the category the PR falls into create a new file in the respective directory
+with the filename format `DD-MM-YYYY-<few-words-about-the-change>.md`
+
+The contents of the file should describe the changes in an ellaborative manner, except for `Updates`.
+As `Updates` refer to the pacakage updates, the description of the file should be of the
+following format: `Package Name [Version](link to changelog)`
```

Comment:
I would like to see an example of it. I get the `Package Name` part. It would be something like `dev-util/perf`. `Version` would be version taken from ebuild filename, right (so `5.15-r1`)? I don't get the `link to changelog` part - what changelog?

Reply:
It means the release notes of the updated package

Reply:
Could be tricky for some projects. What would use for `dev-util/perf`? https://kernelnewbies.org/Linux_5.15#Tracing.2C_perf_and_BPF for example?

Reply:
Package name is the upstream name, no need to prefix the group. Also, version is the upstream release version w/o recipe suffix. Changelog is a link to the upstream release changelog (if available).
A good example would be previously updated item from an existing changelog, e.g. Linux in the latest stable release:

```
# Updates
- Linux ([5.10.77](https://lwn.net/Articles/874852/))
```

Reply:
Cool, thanks for the example. I think it needs to be clarified in the README.